pkg/clients/db_client.go

```diff
@@ -33,6 +33,8 @@ func NewDBClient(host, user, password string, port int, database, application_na
 }
 
 func buildConnectionString(host, user, password string, port int, database, sslmode, application_name string) string {
+	options := "--transaction_isolation=strict\\ serializable"
```

Comment:
Embarrassed to admit how long it took me to get this syntax correct 😁

Reply:
The backslashes will continue until morale improves :laughing: